file2.js

```diff
@@ -0,0 +1,558 @@
+      });
+    };
+
+
+
+    ProjectSettings.prototype.bindPrReviewTimeDropdown = function(projectId) {
+      $(".BtnUpdateTimeToReviewPR").click(function() {
+        var timeToReviewPR = $(".TxtUpdateTimeToReviewPR").val();
+        var url = $(this).data("url").replace(-1, timeToReviewPR);
+        var postVariable = {};
+
+        utils.post(url, postVariable, function(e) {
+          if (e.error) {
+            $(".TimeToReviewPRFeedback").showError(e.error);
+          } else {
+            $(".TimeToReviewPRFeedback").showSuccess("Saved");
+            location.reload();
+          }
+        }, function() {
+          $(".TimeToReviewPRFeedback").showError("Could not save name, please try again.");
+        });
+      });
+
+      $(".TxtUpdateTimeToReviewPR").keyup(function() {
+        $(".BtnUpdateTimeToReviewPR").removeClass("disabled");
+      });
+    };
+
+    var setPushHook = function(isIntegrationActive, isHookActive, activateUrl, deactivateUrl) {
+      if (isIntegrationActive) {
+        $("#hookToggle").show();
+        $("#integrationNotEnabled").hide();
+        $(".hook-setup-automatic").show();
+        $("#hook-collapse").collapse('hide');
+        $('#hookFeedback').hide();
+        $("#hookToggle").click(function() {
+          var $this = $(this);
+          var url = '';
+          if (isHookActive) {
+            url = deactivateUrl;
+          } else {
+            url = activateUrl;
+          }
+          utils.JSend(url, {}, function() {
+            setPushHook(isIntegrationActive, !isHookActive, activateUrl, deactivateUrl);
+            var indicator = $this.closest(".notification-item").find(".notification-indicator");
+            indicator.removeClass("circle-red").addClass("circle-green");
+          }, function() {
+            $this.prop("checked", false);
+            if (url === activateUrl) {
+              $('#hookFeedback').show();
+              $("#hook-collapse").collapse('show');
+            } else {
+              $('#hookFeedbackOnRemove').show();
+            }
+          });
+        });
+      } else {
+        $('#hookFeedback').hide();
+        $("#hookToggle").hide();
+        $("#integrationNotEnabled").show();
+        $(".hook-setup-automatic").hide();
+        $("#hook-collapse").collapse('show');
+      }
+    };
+
+    ProjectSettings.prototype.bindPushHook = function(isIntegrationActive, isHookActive, activateUrl, deactivateUrl) {
+      setPushHook(isIntegrationActive, isHookActive, activateUrl, deactivateUrl);
+    };
+
+    ProjectSettings.prototype.bindOutsideCollaborators = function() {
+      $(".js-content-edit").click(function(e) {
+        e.preventDefault();
+        $(".co__popover").toggle();
+      });
+
+      var $inputEmail = $("#js-add-outside-collaborator-email");
+      var $addCollaborator = $(".js-add-outside-collaborator");
+
+      $inputEmail.keyup(function(e) {
+        $(this).parent().removeClass("has-error");
+
+        if (e.keyCode == 13) {
```

Comment:
![Codacy](https://app.dev.codacy.org/assets/images/favicon.png) Issue found: [Expected '===' and instead saw '=='. (eqeqeq)](https://app.dev.codacy.org/app/codacytestuser/CheckPrCommentsPublicProject/pullRequest?prid=7992)